internal/testdata/sampleHistory.json

```diff
@@ -0,0 +1,46 @@
+[
```

Comment:
Is this the format produced by UI?

Reply:
It's the one produced by downloading the file from the cli

Reply:
The UI format is different than what's produced by the api. This is tracked by https://github.com/uber/cadence-web/issues/19